pkg/scheduler/scheduler.go

```diff
@@ -103,6 +104,10 @@ type Scheduler struct {
 	// Recorder is the EventRecorder to use
 	Recorder events.EventRecorder
```

Comment:
same question... a v1 Events implementation of the EventRecorder interface would make more sense